### DISCUSSION TITLE
Link for 06/05 Monday meeting

### DISCUSSION THREAD

**Discussion by ZIYANGSONG2003** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#88] [Announcements]
Stephanie is inviting you to a scheduled Zoom meeting.

Topic: Stephanie's Zoom Meeting
Time: May 6, 2024 03:00 PM Canberra, Melbourne, Sydney

Join Zoom Meeting
https://anu.zoom.us/j/86540050909?pwd=Z0FnZWc0UW5kRndQN3lFTm40WGF1Zz09

Meeting ID: 865 4005 0909
Password: 677658
One tap mobile
+12532158782,,86540050909#,,,,0#,,677658# US (Tacoma)
+13017158592,,86540050909#,,,,0#,,677658# US (Washington DC)

Dial by your location
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 305 224 1968 US
        +1 309 205 3325 US
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 360 209 5623 US
        +1 386 347 5053 US
        +1 507 473 4847 US
        +1 564 217 2000 US
        +1 646 876 9923 US (New York)
        +1 646 931 3860 US
        +1 669 444 9171 US
        +1 669 900 6833 US (San Jose)
        +1 689 278 1000 US
        +1 719 359 4580 US
        +1 253 205 0468 US
Meeting ID: 865 4005 0909
Password: 677658
Find your local number: https://anu.zoom.us/u/kXFCpfN0T

Or an H.323/SIP room system:
    Dial: 86540050909@global.zoomcrc.com
    Meeting ID: 86540050909
    H323/SIP Password: 677658

**Comment by 2Jus2** [comment]
see u guys in this meeting 
In this week, we are going to discuss about Audit 3 
- What needs to be involved?
- Distribute the work
